lib/api/v1/forms/upload.go

```diff
@@ -0,0 +1,88 @@
+package forms
+
+import (
+	"errors"
+	"mime/multipart"
+
+	"go-google-scraper-challenge/helpers"
+	"go-google-scraper-challenge/lib/models"
+
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+type UploadForm struct {
+	File       multipart.File
+	FileHeader *multipart.FileHeader
+	UserID     int64
```

Comment:
Should we pass this `UserID` as a `User` instead?
The benefit of using an object here are:
- Using with the gorm's association when updating it to the database
- Validating user before save

----

If it takes times, let's keep it like this I'm OK 🙌

Reply:
Good idea, updated in [df1ab4f](https://github.com/carryall/go-google-scraper-challenge/pull/97/commits/df1ab4f70233399770ab4fefa0971db702fa9298) along with the comment below